actiontext/test/template/form_helper_test.rb

```diff
@@ -28,19 +28,39 @@ def form_with(*, **)
   test "rich text area tag" do
```

Comment:
I think it would make sense to make this test name match the format of the subsequent `"#rich_text_area helper"` test as well:

```suggestion
  test "#rich_text_area_tag helper" do
```